mobilecoind-dev-faucet/README.md

```diff
@@ -6,6 +6,9 @@ This is a standalone http server which provides faucet functionality.
 * Any token id can be requested for testing
 * TODO: HTTP Authorization headers may be added in the future
 
+The faucet also supports network load-testing functionality called "slam".
```

Comment:
I'm concerned about the name becoming misleading. How about we rename this something like `mobilecoin-dev-helper`?

Reply:
`mobilecoind-slamming-faucet`? I find `-helper` meaningless in this context. What does it helps the devs with?
(I am also fine leaving it as `dev-faucet`)

Reply:
I can't think of a better name, so I guess `dev-faucet` works well enough.